src/piadm.sh

```diff
@@ -239,6 +239,7 @@ install() {
 		# URL_PREFIX.  Grab the latest-version ISO. Before proceeding,
 		# make sure it's the current one.
 		iso=yes
+		vecho "Downloading latest SmartOS ISO"
 		"${CURL[@]}" -o "${tdir}/smartos.iso" "${URL_PREFIX}/smartos-latest.iso"
```

Comment:
Since we're being verbose now, consider adding `--progress-bar` to curl

Reply:
See below...